docs_app/package.json

```diff
@@ -44,94 +44,94 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^6.1.0",
-    "@angular/cdk": "^6.4.0",
-    "@angular/common": "^6.1.0",
-    "@angular/compiler": "^6.1.0",
-    "@angular/core": "^6.1.0",
-    "@angular/elements": "^6.1.0",
-    "@angular/forms": "^6.1.0",
-    "@angular/material": "^6.4.0",
-    "@angular/platform-browser": "^6.1.0",
-    "@angular/platform-browser-dynamic": "^6.1.0",
-    "@angular/router": "^6.1.0",
-    "@angular/service-worker": "^6.1.0",
+    "@angular/animations": "^8.0.0",
+    "@angular/cdk": "^8.0.1",
+    "@angular/common": "^8.0.0",
+    "@angular/compiler": "^8.0.0",
+    "@angular/core": "^8.0.0",
+    "@angular/elements": "^8.0.0",
+    "@angular/forms": "^8.0.0",
+    "@angular/material": "^8.0.1",
+    "@angular/platform-browser": "^8.0.0",
+    "@angular/platform-browser-dynamic": "^8.0.0",
+    "@angular/router": "^8.0.0",
+    "@angular/service-worker": "^8.0.0",
     "@stackblitz/sdk": "^1.2.0",
-    "@webcomponents/custom-elements": "^1.0.8",
+    "@webcomponents/custom-elements": "^1.2.4",
     "classlist.js": "^1.1.20150312",
-    "core-js": "^2.4.1",
-    "rxjs": "^6.2.0",
-    "tslib": "^1.9.0",
-    "web-animations-js": "^2.2.5",
-    "zone.js": "^0.8.26"
+    "core-js": "^3.1.3",
+    "rxjs": "^6.5.2",
+    "tslib": "^1.10.0",
+    "web-animations-js": "github:angular/web-animations-js#release_pr208",
+    "zone.js": "~0.9.1"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.7.4",
-    "@angular/cli": "^6.1.0",
-    "@angular/compiler-cli": "^6.1.0",
-    "@types/jasmine": "^2.5.52",
+    "@angular-devkit/build-angular": "~0.800.0",
+    "@angular/cli": "^8.0.2",
```

Comment:
This upgrade requires Node.js 10.9 or greater:

>You are running version v8.15.1 of Node.js, which is not supported by Angular CLI 8.0+.
>The official Node.js version that is supported is 10.9 or greater.

(after running `npm start`)

`node` property must be updated accordingly under `engines`.

Reply:
Thanks for the feedback and double checking the code!